packages/gestalt/src/utils/datavizcolors.js

```diff
@@ -0,0 +1,82 @@
+// @flow strict
+import { type Theme } from '../contexts/ColorSchemeProvider.js';
+import { type InteractionStates, type ColorStyles } from '../Tile/Tile.js';
+import { type DataVisualizationColors } from '../TileData.js';
+
+export default class DataVizColor {
```

Comment:
@dangerismycat - not sure what happened to your old comment. I renamed this file, so maybe it got lost.

Re: helper methods being separate files. In this particular case, all these methods are fairly tightly coupled. So if I were break them out, should I toss them in a folder and add an index.js to import them as one? I feel like I'd like them to live in one file at the very least, if not in a class.

The point of the class is really just to group these things together and to have autocomplete say `Datavizcolor.(showmethodname)`. They are static functions. We're not using constructors or anything.

Here's an example of this pattern in the [VSCode](https://github.com/microsoft/vscode/blob/db6bcdf309cc1565c502a54b73c33a63bdabfbd8/src/vs/css.build.ts#L166) Codebase. 

Lmk your thoughts

Reply:
1) When working in a codebase that's new to you, take a look around and try to make your code conform to the general style of the existing code. We don't use OOP anywhere in Gestalt. That isn't by accident.

2) Classes are an unnecessary abstraction for utils. If you want to use a single util function, you have to import the entire class. That's a lot of extra imported code for no benefit.

3) Single-function util functions within a utils directory make identifying dead code much easier.

4) Index files are an unnecessary abstraction that make stack traces more difficult to read. Modern code editors have good autocomplete — import only the things you need where you're using them.